tests/queries/0_stateless/02346_text_index_function_hasAnyAllTokens.reference

```diff
@@ -32,15 +59,35 @@ SELECT id FROM tab WHERE hasAnyTokens(col_str, ['b']);
 1
 SELECT id FROM tab WHERE hasAnyTokens(col_str, ['c']);
 2
+SELECT id FROM tab WHERE hasAnyTokens('a b', 'b');
+1
+2
+3
+SELECT id FROM tab WHERE hasAnyTokens('a b', 'c');
+SELECT id FROM tab WHERE hasAnyTokens(col_str, 'b');
+1
+SELECT id FROM tab WHERE hasAnyTokens(col_str, 'c');
+2
 SELECT id FROM tab WHERE hasAllTokens('a b', ['a b']);
 SELECT id FROM tab WHERE hasAllTokens('a b', ['a c']);
 SELECT id FROM tab WHERE hasAllTokens(col_str, ['a b']);
 SELECT id FROM tab WHERE hasAllTokens(col_str, ['a c']);
+SELECT id FROM tab WHERE hasAllTokens('a b', 'a b');
+1
+2
+3
+SELECT id FROM tab WHERE hasAllTokens('a b', 'a c');
+SELECT id FROM tab WHERE hasAllTokens(col_str, 'a a');
+SELECT id FROM tab WHERE hasAllTokens(col_str, 'b c');
 -- Test search without needle on non-empty columns (all are expected to match)
 SELECT count() FROM tab WHERE hasAnyTokens(col_str, []);
 3
 SELECT count() FROM tab WHERE hasAllTokens(col_str, []);
 3
+SELECT count() FROM tab WHERE hasAnyTokens(col_str, ['']); -- matches nothing
+0
+SELECT count() FROM tab WHERE hasAnyTokens(col_str, ''); -- TODO currently this goes through the default tokenizer and matches everything
+3
```

Comment:
Need to decide here what functionality we want and whether it should be the same between the two overloads or of this difference makes sense.